django/utils/translation/trans_real.py

```diff
@@ -378,7 +378,7 @@ def all_locale_paths():
     """
     globalpath = os.path.join(
         os.path.dirname(upath(sys.modules[settings.__module__].__file__)), 'locale')
-    return [globalpath] + list(settings.LOCALE_PATHS)
+    return [globalpath] + (settings.LOCALE_PATHS)
```

Comment:
Actually, we should probably keep the list() call for compatibility with tuples.